### DISCUSSION TITLE
Partial clearing of  cache from redis for a given hash

### DISCUSSION THREAD

**Discussion by sunilchaurha** [redisson/redisson#4622] [Q&A]
Is there a way in redisson to partial clear cache for a given hash. Lets say I cache data for multiple dates and I want to clear for one specific date.
Is it possible easily?
@mrniko

**Comment by mrniko** [comment]
Do you mean partially clear local cache?

**Comment by sunilchaurha** [comment]
Not local cache, I want to clear from redis cluster (elasticache - AWS).

On Fri, 21 Oct 2022 at 20:04, Nikita Koksharov ***@***.***>
wrote:

> Do you mean partially clear local cache?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/redisson/redisson/discussions/4622#discussioncomment-3934332>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7SMKXPD43UP6WECGAMNZTWEKSZFANCNFSM6AAAAAARLG2GRY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

--------------
Best Regards,
Sunil  Chauraha

**Reply by mrniko** [reply]
Use RMap.fastRemove method
